Foundation/URLSession/libcurl/MultiHandle.swift

```diff
@@ -217,9 +217,14 @@ fileprivate extension URLSession._MultiHandle {
         // Find the NSURLError code
         var error: NSError?
         if let errorCode = easyHandle.urlErrorCode(for: easyCode) {
-            let errorDescription = easyHandle.errorBuffer[0] != 0 ?
-                String(cString: easyHandle.errorBuffer) :
-                unsafeBitCast(CFURLSessionCreateErrorDescription(easyCode.value), to: NSString.self) as String
+            var errorDescription: String = ""
+            if easyHandle.errorBuffer[0] == 0 {
+              let description = CFURLSessionEasyCodeDescription(easyCode)
+              errorDescription = NSString(bytesNoCopy: UnsafeMutableRawPointer(mutating: description), length: strlen(description), encoding: String.Encoding.utf8.rawValue, freeWhenDone: false)! as String
```

Comment:
This violates the implied contract of bridging, where we reserve the right to keep around the object you’re trying to bridge. It happens to work in SCF but it may or may not going forward, and would’ve crashed on Darwin for sure.

Reply:
(Just drop the `…NoCopy`.)

Reply:
That is a good point, better to be safe and do the copy.  The reason that I had elided the copy is because `curl_easy_strerror` would return a string that is globally allocated and can be preserved.